integration-tests/bats/log.bats

```diff
@@ -632,3 +632,15 @@ teardown() {
     run dolt commit --allow-empty -m "% should be escaped"
     [[ "$output" =~ "% should be escaped" ]] || false
 }
+
+@test "log: identify HEAD" {
+    dolt commit --allow-empty -m "commit 1"
+    dolt tag commit1
+    dolt commit --allow-empty -m "commit 2"
+    dolt tag commit2
+    run dolt log commit1
+    [[ !("$output" =~ "HEAD") ]] || false
+    dolt log commit2 | cat
+    run dolt log commit2 | cat
```

Comment:
Do you need the `| cat`? `run` doesn't work well with pipes, so I'm guessing this isn't doing what it looks like:
https://bats-core.readthedocs.io/en/stable/writing-tests.html#run-and-pipes

Reply:
That was leftover from when I was trying to get the test to suppress ANSI color codes, but I don't think we've implemented that yet. I just changed the test to not need it. Removed.